linera-core/src/worker.rs

```diff
@@ -1101,7 +1113,10 @@ where
                 duplicated = d;
                 (info, actions)
             }
-            CertificateValue::ConfirmedBlock { .. } => {
+            CertificateValue::ConfirmedBlock { executed_block } => {
+                confirmed_transactions = (executed_block.block.incoming_messages.len()
+                    + executed_block.block.operations.len())
+                    as u64;
```

Comment:
Should this be here or after `process_confirmed_block`?

Reply:
Doesn't matter; if `process_confirmed_block` returns an error we won't increase the counter anyway.